test/Crate/Test/DBAL/Functional/Schema/SchemaManagerTest.php

```diff
@@ -75,16 +76,17 @@ public function testListTables()
     public function createListTableColumns()
     {
         $table = new Table('list_table_columns');
-        $table->addColumn('text', Type::STRING);
+        $table->addColumn('text', Types::STRING);
         $table->addColumn('ts', TimestampType::NAME);
-        $table->addColumn('num_float_double', Type::FLOAT);
-        $table->addColumn('num_short', Type::SMALLINT);
-        $table->addColumn('num_int', Type::INTEGER);
-        $table->addColumn('num_long', Type::BIGINT);
+        $table->addColumn('num_float_double', Types::FLOAT);
+        $table->addColumn('num_short', Types::SMALLINT);
+        $table->addColumn('num_int', Types::INTEGER);
+        $table->addColumn('num_long', Types::BIGINT);
         $table->addColumn('id', 'integer', array('notnull' => true));
         $table->setPrimaryKey(array('id'));
 
         // OBJECT schema definition via platform options
+        // Those intentionally use the `Type::getType()` notation and resolve to DBAL types.
```

Comment:
can we add "why/reason"?

Reply:
The reason is because they **do** accept type name literals, while the ctor **does not**. I think I've explained it here:

> // Note that DBAL3 no longer accepts string constants on its `Column` ctor,
> // so use the `Type::getType()` notation instead when defining columns
> // using `new Column(...)`. **Also note that the `addColumn` method still DOES**
> // **accept type names.**

Please suggest any improvements.